src/server_manager/ui_components/outline-server-settings.html

```diff
@@ -133,7 +161,11 @@ <h3>DigitalOcean</h3>
             <h3>[[localize('settings-server-info')]]</h3>
             <paper-input id="serverNameInput" class="server-name" value="{{name}}" label="[[localize('settings-server-name')]]" always-float-label maxlength="100" on-keydown="_handleNameInputKeyDown" on-blur="_handleNameInputBlur"></paper-input>
             <p class="detail">[[localize('settings-server-rename')]]</p>
-            <paper-input readonly value="[[serverPortForNewAccessKeys]]" label="[[localize('settings-access-key-port')]]" hidden$="[[!serverPortForNewAccessKeys]]" always-float-label maxlength="100"></paper-input>
+            <div id="portForNewAccessKeysContainer">
+              <paper-input id="keysPortInput" value="[[serverPortForNewAccessKeys]]" label="[[localize('settings-access-key-port')]]" maxlength="5" error-color="#ffffff" error-message="ERRORMESSSAGE" hidden$="[[!serverPortForNewAccessKeys]]" on-click="_handleKeysPortClick" on-keyup="_handleKeysPortInputKeyDown" always-float-label maxlength="100"></paper-input>
```

Comment:
Do we want this: `message="ERRORMESSAGE"`?

Reply:
It never gets displayed, since the error states all set their own message.  Changed to an empty string

Reply:
We actually don't need the `error-message` attribute. Please remove it.

Reply:
Done